recipes/zamp/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "zamp" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/metagenlab/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 366d1a1bdff60e1a8100298608f20ab2294ef476a1da1bee9c1975641c495f6e
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - zamp=zamp.__main__:main
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation --no-cache-dir -vvv
+  run_exports:
+    - {{ pin_subpackage('zamp', max_pin="x.x") }}
```

Comment:
Usually versions `1.+` should use `max_pin="x"`. Are you sure about `"x.x"` ?
See https://github.com/bioconda/bioconda-recipes/blob/master/.github/PULL_REQUEST_TEMPLATE.md#instructions-for-avoiding-api-abi-and-cli-breakage-issues

Reply:
Thank you. I modified meta.yaml to max_pin="x" as suggested.